src/main/scala/rocket/RocketCore.scala

```diff
@@ -943,7 +943,7 @@ class Rocket(tile: RocketTile)(implicit p: Parameters) extends CoreModule()(p)
   xrfWriteBundle.valid := false.B
   xrfWriteBundle.pc := 0.U
   xrfWriteBundle.wrdst := rf_waddr
-  xrfWriteBundle.wrenx := rf_wen && !(coreMonitorBundle.wrenx && (rf_waddr === coreMonitorBundle.wrdst))
+  xrfWriteBundle.wrenx := rf_wen && !(csr.io.trace(0).valid && wb_wen && (wb_waddr === coreMonitorBundle.wrdst))
```

Comment:
`(wb_waddr === coreMonitorBundle.wrdst)` will allows be `true`

Reply:
Oh damn. I meant `(rf_waddr === wb_addr)` I'll make this change